lib/ZumoOta/src/MySettings.h

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+    DESCRIPTION
+*******************************************************************************/
+/**
+ * @brief  ZumoOta application
+ * @author Decareme Pauline Ngangnou <ngandeca@yahoo.fr>
+ * @{
+ */
+#ifndef MySettings_H
+#define MySettings_H
+
+/******************************************************************************
+ * Includes
+ *****************************************************************************/
+
+/******************************************************************************
+ * Macros
+ *****************************************************************************/
+ 
+/******************************************************************************
+ * Types and Classes
+ *****************************************************************************/
+
+class MySettings{
+private:

```

Comment:
check indentation of file.  line 56+ looks right.

Please pay more attention to such formal things. It is a bit tiring for reviewer to  complain about such easy to avoid things.

Reply:
It will be performed by the next commit.